src/components/button/AposButton.vue

```diff
@@ -60,8 +67,16 @@ export default {
       return (this.disabled || this.busy) ? true : false;
     }
   },
+  watch: {
+    busy (val) {
```

Comment:
This is where I am less sure. Watchers should generally be avoided, but this seems like a decent use case. I don't think computed would work.

Reply:
Not "avoided," maybe, but not overused.

Reply:
Not avoided, necessarily, but not overused. Which I don't think this is doing.